js/utils.js

```diff
@@ -544,7 +544,8 @@ Fliplet.Registry.set('dynamicListUtils', (function () {
 
     return Promise.resolve({
       records: searchResults,
-      truncated: truncated
+      truncated: truncated,
+      filteredResult: showBookmarks || !_.isEmpty(activeFilters) || value
```

Comment:
Cast this into a boolean value instead of having to cast it elsewhere. 

```suggestion
      filteredResult: !!(showBookmarks || !_.isEmpty(activeFilters) || value)
```

Reply:
Fixed